src/main/resources/spark-swagger.conf

```diff
@@ -19,12 +19,14 @@ spark-swagger {
   # API related configs
 
   # Location where API docs will be published
+  # PublicApiDocGenerator will overwrite this for the public-facing swagger document.
   host = "localhost:4567"
 
-  # basepath for calling the API, if applicable, otherwise leave as "/"
-  # as in https://example.api.com/<basepath>/user/all
+  # basePath must be "/" so that endpoints are in the form localhost:4567/endpoints.
+  # PublicApiDocGenerator will overwrite this for the public-facing swagger document.
+  # (Enter a path other than "/" to make if you want all endpoints to be in the form localhost:4567/basepath/endpoints.)
```

Comment:
There are two comments about `PublicApiDocGenerator will overwrite this`, but I have no idea what "this" is.

Reply:
Updated in https://github.com/ibi-group/otp-middleware/pull/59/commits/b327a799b261096bd67fcaf0d680d92b1090fc2a